### ISSUE TITLE
[object_detection]tensorflow.python.framework.errors_impl.UnknownError

### ISSUE THREAD

**Issue by dapsjj** [tensorflow/models#8017]
**System information**
**What is the top-level directory of the model you are using:** models-master/research/object_detection/
**Have I written custom code (as opposed to using a stock example script provided in TensorFlow):** Yes
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):** Windows10
**TensorFlow installed from (source or binary):** binary
**TensorFlow version (use command below):** 1.12.0
**Bazel version (if compiling from source):** No
**CUDA/cuDNN version:**  No
**GPU model and memory:** GTX 1050 4G/memory 16G 
**Exact command to reproduce:**  No

**Problem description**：
When I use this command to train:

> python model_main.py --pipeline_config_path=training/ssd_inception_v2_coco.config --model_dir=training/ --num_train_steps=200000  --alsologtostderr

The error message is:

> model_main.py 109 <module>
> tf.app.run()
> 
> app.py 125 run
> _sys.exit(main(argv))
> 
> model_main.py 105 main
> tf.estimator.train_and_evaluate(estimator, train_spec, eval_specs[0])
> 
> training.py 471 train_and_evaluate
> return executor.run()
> 
> training.py 610 run
> return self.run_local()
> 
> training.py 711 run_local
> saving_listeners=saving_listeners)
> 
> estimator.py 354 train
> loss = self._train_model(input_fn, hooks, saving_listeners)
> 
> estimator.py 1207 _train_model
> return self._train_model_default(input_fn, hooks, saving_listeners)
> 
> estimator.py 1241 _train_model_default
> saving_listeners)
> 
> estimator.py 1471 _train_with_estimator_spec
> _, loss = mon_sess.run([estimator_spec.train_op, estimator_spec.loss])
> 
> monitored_session.py 671 run
> run_metadata=run_metadata)
> 
> monitored_session.py 1156 run
> run_metadata=run_metadata)
> 
> monitored_session.py 1255 run
> raise six.reraise(*original_exc_info)
> 
> six.py 696 reraise
> raise value
> 
> monitored_session.py 1240 run
> return self._sess.run(*args, **kwargs)
> 
> monitored_session.py 1312 run
> run_metadata=run_metadata)
> 
> monitored_session.py 1076 run
> return self._sess.run(*args, **kwargs)
> 
> session.py 929 run
> run_metadata_ptr)
> 
> session.py 1152 _run
> feed_dict_tensor, options, run_metadata)
> 
> session.py 1328 _do_run
> run_metadata)
> 
> session.py 1348 _do_call
> raise type(e)(node_def, op, message)
> 
> tensorflow.python.framework.errors_impl.UnknownError
>

**Comment by gadagashwini** [comment]
Hi @dapsjj, Could you try the workarounds mentioned on similar issue [#4177](https://github.com/tensorflow/models/issues/4177#issuecomment-733579398). Looks like issue with checkpoints folder, check access to folder where checkpoints are stored. Thank you!
